cmake/Buildevent-driven.cmake

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2020  iCub Facility, Istituto Italiano di Tecnologia
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+include(FindOrBuildPackage)
+
+find_or_build_package(YARP QUIET)
```

Comment:
As event-driven depends on YCM (see https://github.com/robotology/event-driven/blob/master/CMakeLists.txt#L28) I suggest to add the dependency on YCM .

Reply:
Your're right, I just added that dependency. Thanks.